lib/workers/repository/reconfigure/comment.ts

```diff
@@ -0,0 +1,123 @@
+import is from '@sindresorhus/is';
+import { GlobalConfig } from '../../../config/global';
+import type { RenovateConfig } from '../../../config/types';
+import { logger } from '../../../logger';
+import type { PackageFile } from '../../../modules/manager/types';
+import type { Pr } from '../../../modules/platform';
+import { platform } from '../../../modules/platform';
+import { ensureComment } from '../../../modules/platform/comment';
+import { emojify } from '../../../util/emoji';
+import type { BranchConfig } from '../../types';
+import {
+  getDepWarningsOnboardingPR as getDepsWarnings,
```

Comment:
why renaming? don't do it if not needed.

Reply:
to prevent confusion